### PR TITLE
ltp: Add option to install selected parts from git

### PR DESCRIPTION
**Pull request by czerw**
Fix poo#103608: LTP package is built on daily basis. It can happen
that kernel modules are built against different kernel, than is in
tested snapshot. Variable LTP_GIT_URL allows to install
specific parts from git directly on SUT after was LTP installed from
package.

- Related ticket: https://progress.opensuse.org/issues/103608
- Needles: 
- Verification run: 
install: https://openqa.suse.de/tests/7802590

before: https://openqa.suse.de/tests/7794772#step/insmod01_sh/8
after: https://openqa.suse.de/tests/7802601#step/insmod01_sh/8

### CODE REVIEW

File: tests/kernel/install_ltp.pm [hunk 1]
```diff
@@ -344,6 +366,10 @@ sub run {
     else {
         add_ltp_repo;
         install_from_repo();
+        if (get_var("LTP_INSTALL_SELECTED")) {
```
Comment (mdoucha) [inline tests/kernel/install_ltp.pm:369]: LGTM, but I think there's no need to add a new variable. It'd be better to check `LTP_GIT_URL` instead.

Comment (mdoucha) [reply tests/kernel/install_ltp.pm:369]: Ideally, it'd be better to build the modules from LTP source package for version stability reasons but this is good enough for now.

Comment (czerw) [reply tests/kernel/install_ltp.pm]: Ok changed to `LTP_GIT_URL`. This problem is only for for tests using daily ltp package builds, so fixed version is not needed.

**Review by mdoucha — APPROVED** [review]
LGTM

File: tests/kernel/install_ltp.pm [hunk 2]
```diff
@@ -186,6 +185,29 @@ sub install_from_git {
...
+    my @paths = qw(commands/insmod
+      kernel/firmware
+      kernel/device-drivers
+      kernel/syscalls/delete_module
+      kernel/syscalls/finit_module
+      kernel/syscalls/init_module);
+
+    assert_script_run('pushd testcases');
+    foreach (@paths) {
+        assert_script_run("pushd $_ && make && make install && popd", timeout => 600);
```
Comment (pevik) [inline tests/kernel/install_ltp.pm:201]: nit: Instead `pushd` and `popd` I'd prefer `cd DIR` and `cd -`, but that's really personal (not asking for removing it).

**Review by pevik — APPROVED** [review]
LGTM, thanks!

**Pull request merged**